client/state/ui/editor/post/actions.js

```diff
@@ -0,0 +1,65 @@
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:ui:editor:post:actions' );
+
+export function setEditingMode( mode, modeTitle, site ) {
+	debug( 'setEditingMode', mode, modeTitle, site );
+	return { type: 'TODO' };
+}
+
+export function startEditingNew( site, postOptions ) {
+	debug( 'startEditingNew', site, postOptions );
+	return { type: 'TODO' };
+}
+
+export function startEditingExisting( site, postID ) {
+	debug( 'startEditingExisting', site, postID );
+	return { type: 'TODO' };
```

Comment:
Note that the original fires 'START_EDITING_POST' and also performs an ajax call, where 'RECEIVE_POST_TO_EDIT' fires on successful callback.

I know the you're mirroring the original name, but I wonder if we could improve on this later to something like `loadPostAndTrackEdits`